installer/project/components/params.xml

```diff
@@ -162,6 +162,18 @@ NOTE: If you want to use the Microsoft Azure SQL Server, you must configure Tomc
         </option>
       </optionList>
     </choiceParameter>
+    <linkParameter>
+      <name>infoUpgradingTov2</name>
+      <title>Pre-installation Requirements - Upgrading from Aggregate v1.x</title>
+      <description>https://docs.opendatakit.org/aggregate-intro</description>
```

Comment:
Why do we have this link? Will it have some message about upgrading?

Reply:
https://docs.opendatakit.org/aggregate-upgrade/